### ISSUE TITLE
Diving48 implementation details

### ISSUE THREAD

**Issue by swathikirans** [chenxuluo/GST-video#1]
@chenxuluo, 

Congrats on the ICCV acceptance and thanks for the repo. I have been trying to implement GST on Diving48 dataset. I used the default settings in the repo with 16 segments. So far I am getting only 33% recognition accuracy. I checked the paper and there is no specific details given for training on Diving48 dataset. I would like to know if any of the configuration during training or evaluation is different for this dataset. It would also be helpful if you can update the repo with the code for training on Diving48 dataset including frame extraction and train/val split file generation and the trained models.

**Comment by chenxuluo** [comment]
I trained on 2 GPUs with a batch size of 12 and so half the learning rate. The network was trained with an initial learning rate of 0.005 for about 25 epochs and then decreased by a factor of 10.

**Comment by swathikirans** [comment]
Thank you for the reply. The network is trained for 60 epochs, right? The accuracy reported in the paper is the accuracy obtained at the final epoch of training or the best accuracy obtained across all the epochs? Will you be sharing the weights of the network trained on Diving48 dataset as well?

**Comment by chenxuluo** [comment]
Actually, I trained for much fewer epochs I remembered. The accuracy reported is the best one. I will share the weights later.

**Comment by swathikirans** [comment]
Is the Inference done using 16 segments, single clip and center crop?

**Comment by chenxuluo** [comment]
yes, the reported results were using 16 segments, center crop and sampling.
Here are the pretrained  models: https://drive.google.com/drive/folders/1GjTrSJdlR6i57nK_aVoL2M5SOXZbFa6T?usp=sharing
Actually, the only parameter I tuned is the learning rate schedule.

**Comment by swathikirans** [comment]
Thank you for the models. I am getting very low evaluation performance. I assume it could be because of the way in which I extracted the frames. It would be really helpful if you could either share the frames or the code used to extract the frames from the videos.

**Comment by chenxuluo** [comment]
I just used cv2.VideoCapture to load videos and .read() to extract each frame.
